tests/DbaaS/verifyDBaaSPXCCluster_test.js

```diff
@@ -299,3 +299,77 @@ Scenario('PMM-T717 Verify insufficient resources warning @dbaas @nightly',
     await dbaasActionsPage.verifyInsufficientResources(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.resourceBarMemory, 'Insufficient Memory');
     await dbaasActionsPage.verifyInsufficientResources(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.resourceBarDisk, 'Insufficient Disk');
   });
+
+Scenario('PMM-T704 PMM-T772 PMM-T849 PMM-T850 Resources, PV, Secrets verification @dbaas',
+  async ({
+    I, dbaasPage, dbaasAPI, dbaasActionsPage, adminPage,
+  }) => {
+    const pxc_resource_check_cluster_name = 'pxc-resource-1';
+    let output;
+    const pxc_configuration = {
+      topology: 'Cluster',
+      numberOfNodes: '1',
+      resourcePerNode: 'Custom',
+      memory: '1 GB',
+      cpu: '1',
+      disk: '3 GB',
+      dbType: 'MySQL',
+      clusterDashboardRedirectionLink: dbaasPage.clusterDashboardUrls.pxcDashboard(
+        pxc_resource_check_cluster_name,
+      ),
+    };
+
+    if (!await dbaasAPI.apiCheckRegisteredClusterExist(clusterName)) {
+      await dbaasAPI.apiRegisterCluster(process.env.kubeconfig_minikube, clusterName);
+    }
+
+    await dbaasAPI.deleteAllDBCluster(clusterName);
+    await dbaasPage.waitForDbClusterTab(clusterName);
+    I.waitForDetached(dbaasPage.tabs.kubernetesClusterTab.disabledAddButton, 30);
+    await dbaasActionsPage.createClusterAdvancedOption(clusterName, pxc_resource_check_cluster_name, 'MySQL', pxc_configuration);
+    I.click(dbaasPage.tabs.dbClusterTab.createClusterButton);
+    I.waitForText('Processing', 30, dbaasPage.tabs.dbClusterTab.fields.progressBarContent);
+    await dbaasPage.postClusterCreationValidation(pxc_resource_check_cluster_name, clusterName);
+    const {
+      username, password, host, port,
+    } = await dbaasAPI.getDbClusterDetails(pxc_resource_check_cluster_name, clusterName);
+
+    output = await I.verifyCommand(
```

Comment:
is there a need to assign it to a variable ?

Reply:
no

Reply:
will change it, good point